src/shares/PaymentHub.sol

```diff
@@ -46,14 +49,71 @@ contract PaymentHub {
     
     IQuoter constant uniswapQuoter = IQuoter(0xb27308f9F90D607463bb33eA1BeBb41C27CE5AB6);
     ISwapRouter constant uniswapRouter = ISwapRouter(0xE592427A0AEce92De3Edee1F18E0157C05861564);
+    AggregatorV3Interface internal priceFeedCHFUSD;
+    AggregatorV3Interface internal priceFeedETHUSD;
 
-    constructor(address currency_) {
-        currency = currency_;
+    constructor(address _currency, address _aggregatorCHFUSD, address _aggregatorETHUSD) {
```

Comment:
Is it possible that _aggregatorCHFUSD and _aggregatorETHUSD addresses change at some point, or are they as hardcoded as the UniswapV3Router, for example.
Do we pass them as parameters, because they may have different addresses on different networks, like Polygon etc.?